index.js

```diff
@@ -1,11 +1,20 @@
 import express from "express";
 import axios from "axios";
 import bodyParser from "body-parser";
+import { registerUser, signInUser } from './public/js/firebase';    //Firebse Added
+import { registerUser, signInUser, signOutUser } from './public/js/auth'; //imported auth
+import { addData, getData, updateData, deleteData } from './public/js/database';  //imported database
+
+const { database, auth } = require('./public/js/firebase');
```

Comment:
Even if I comment it out reset of the code is still not working.

Reply:
Ohh ok
Sorry for that
IDK why it ran in my machine
I'll just modify again👍

Reply:
> Ohh ok Sorry for that IDK why it ran in my machine I'll just modify again👍

I think you are using a new Javascript runtime which we don't have access to ,cause Node won't be able to run this code